testing/e2e/playwright.config.ts

```diff
@@ -5,7 +5,7 @@ import { defineConfig, devices } from '@playwright/test';
  */
 export default defineConfig({
   testMatch: 'app/**/*spec.ts', // Look for test files anywhere in the app
-
+  timeout: 100000,
```

Comment:
Since the test row number is 100000, it took a while to load the table and the sub-components so I boosted the timeout to 100s for it to work locally. This might be too much so if there is any other better solution, let me know!

Reply:
Can you describe the problem that led you to increase the timeout?

Is there a performance issue that we need to fix? I see in the playground that you're only testing it with a few rows. Could you try to testing it manually with 10_000 or 100_000?

Reply:
I have tested manually with 10,000 and 100,000 and the playground loaded pretty quickly. However, for the playwright test, I kept getting the timeout error until I increased the timeout to 100s, when the test finally passed locally.

Reply:
Interestingly, it passes on my machine without the increased timeout. The UI is ready within a few hundred milliseconds. I understand that my M1 MacBook Pro might be more powerful than the average Windows laptop, but 100 seconds seems excessive.

![](https://github.com/user-attachments/assets/610ed498-2178-4136-b9bb-4638297eb37e)